### DISCUSSION TITLE
Dioxus Web without CSS framework?

### DISCUSSION THREAD

**Discussion by xpe** [DioxusLabs/dioxus#2404] [Q&A] [answered]
I would think it is possible to use Dioxus (use the "Web" web renderer) without any CSS framework, right?

This is what I see when I run `dx new`:
```
% dx new
✔ 🤷   Which sub-template should be expanded? · Web
🤷   Project Name: intage
? 🤷   How do you want to create CSS? ›
  Tailwind
❯ Vanilla
```

**Comment by DogeDark** [comment, marked as answer]
The "Vanilla" option isn't a framework, but rather regular CSS. It will generate a basic app that has a CSS file. To remove it you only need to delete the `main.css` file in the assets folder and remove the `link` item in the RSX.

**Reply by xpe** [reply]
It might sound like a small thing, but making it lowercase `vanilla` would help reduce the ambiguity with [Vanilla JS](http://vanilla-js.com).

**Reply by DogeDark** [reply]
I was wondering if there was a framework called Vanilla. I agree, we could rename it to something else.

**Reply by ealmloff** [reply]
[Vanilla JS](http://vanilla-js.com/) is also not a framework 😆

**Reply by xpe** [reply]
I wish I could have said I was in on the joke... A while back, I remember looking at that page and thinking "haha, nicely done". Somehow I forgot. The joke lives on.
